plugin/src/main/kotlin/com/nishtahir/CargoBuildTask.kt

```diff
@@ -58,12 +58,9 @@ open class CargoBuildTask : DefaultTask() {
                 if (targetIncludes != null) {
                     spec.include(targetIncludes.asIterable())
                 } else {
-                    // It's safe to unwrap, since we bailed at configuration time if this is unset.
-                    val libname = libname!!
-                    spec.include("lib${libname}.so")
-                    spec.include("lib${libname}.dylib")
-                    spec.include("${libname}.dll")
+                    spec.include(libname!!)
                 }
```

Comment:
Huh, is this right for non-linux platforms? macos uses `.dylib` and windows uses `.dll`.

Reply:
Oh maybe the issue is that I am compiling a binary to be bundled into the apk -- not a library.

Reply:
Right, for bundling an APK that's right, but rust-android-gradle also supports being used for other situations (namely unit tests on the host machine, as discussed in comments below).

Sorry for all the trouble here!